includes/ets_admin_qa_function.php

```diff
@@ -484,6 +484,7 @@ public function admin_qa_save( $productId ){
 				$to = $value['user_email'];
 				$userName = $value['user_name'];
 				$productTitle = $value['product_title'];
+				$questions = $value['question'];
```

Comment:
`$questions` does it contain multiple questions? why name is plural.
Remember to change where it is being used as well.